features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayConfiguration.kt

```diff
@@ -76,6 +78,11 @@ data class SessionReplayConfiguration internal constructor(
             return this
         }
 
+        internal fun setTouchPrivacy(level: TouchPrivacy): Builder {
```

Comment:
why internal ?

Reply:
We don't want to make these apis public until all of fgm is completed, however I've decided that it's better that I open a feature branch and merge these prs into that, and then this could be public in that branch